install_cexts.py

```diff
@@ -58,7 +59,7 @@ def download_package(path, platform, version, implementation, abi, name):
     return_code = subprocess.call([
         'python', 'kolibripip.pex', 'download', '-q', '-d', path, '--platform', platform,
         '--python-version', version, '--implementation', implementation,
-        '--abi', abi, '-i', index_url, name
+        '--abi', abi, '-i', index_url, '{}=={}'.format(name, pk_version)
```

Comment:
@lyw07 can you check the merge conflict resolutions in this file to make sure they are right? I erred in favour of 0.6.1 as that seemed to be more recently changed.

Reply:
Yes the second line from 0.6.1 is the correct one! It is to fix the version issue for installing cryptography.